### PR TITLE
Fixed mobile hover state

### LINKED ISSUE #4522: Mobile: Hover effect stays on tapped elements
**Description**:
Mobile: Hover effect stays on tapped elements

**Expected behavior**:
Hover effect isn't displayed on tapped elements

**To Reproduce**:
1. Open PDP in mobile layout
2. Tap '+' symbol
3. Tap "Notify me when the price drops'

**Actual result**:
Hover effect stays on tapped elements

**Additional context**:
The same situation with:
- '+' and '-' symbols in the cart, Wishlist
- 'Notify me when this product is in stock' on PDP
- Samples for downloadable products
- File in My Downloadable

### PR DESCRIPTION
**Pull request by SalimMersally**
**Related issue(s):**
* Fixes #4522 

**Problem:**
* Hover state stay after tapping on mobile view

**In this PR:**
* Fix hover state

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
1. Android: can reproduce 
https://drive.google.com/file/d/1YuMX0SU1kFjvVcEU92OfNKjevBudZ0ZF/view?usp=sharing
2. iOS: + and - stay hovereвю Also, reorder button
https://drive.google.com/file/d/1crHlC4nfGmLv1gA2kpdiGKudP-8dTJkr/view?usp=sharing
3. Don't show hover effect for 'notify...' and samples links in mobile view

**Comment by aleksandrakorolova** [comment]
1. Fixed
2. Still, hover effect stays for Reorder and Subscribe to Order status on Order page

File: packages/scandipwa/src/component/MyAccountOrder/MyAccountOrder.style.scss [hunk 1]
```diff
@@ -45,9 +45,15 @@
         margin-inline-end: 20px;
 
         &.Button {
-            &:hover {
+            &:active {
                 color: var(--link-hover);
             }
+
+            @media (hover: hover) and (pointer: fine) {
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/MyAccountOrder/MyAccountOrder.style.scss:52]: This is a beautiful solution, but too complex to memorize.
Please make a helper mixin for it and replace all occurrences with the new mixin.
As an example, you can see `@desktop` or `@mobile` mixins.

**Review by zans-laksa — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/ProductActions/ProductActions.style.scss [hunk 2]
```diff
@@ -295,10 +295,16 @@
             &:disabled {
                 cursor: default;
             }
-
-            &:hover {
+
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/ProductActions/ProductActions.style.scss:298]: Please remove trailing spaces.

File: packages/scandipwa/src/style/abstract/_media.scss [hunk 3]
```diff
@@ -73,3 +73,9 @@
         @content;
     }
 }
+
+@mixin hovarable {
```
Comment (zans-laksa) [inline packages/scandipwa/src/style/abstract/_media.scss:77]: Please replace hovarable with hoverable.

File: packages/scandipwa/src/style/base/_input.scss [hunk 4]
```diff
@@ -114,13 +114,23 @@ textarea {
...
+            
+        &:active {
             border-color: var(--primary-dark-color);
 
             svg {
                 fill: var(--primary-dark-color);
-            }
-        }
+            }            
+        }
```
Comment (zans-laksa) [inline packages/scandipwa/src/style/base/_input.scss:133]: Please remove trailing spaces.

**Review by zans-laksa — CHANGES_REQUESTED** [review]

**Review by zans-laksa — APPROVED** [review]

**Pull request merged**